ospec/README.md

```diff
@@ -455,14 +463,81 @@ $o("a test", function() {
 $o.run()
 ```
 
+---
+
+## Result data
+
+Test results are available by reference for integration purposes. You
+can use custom reporters in `o.run()` to process these results.
+
+```javascript
+o.run(function (results) {
+	// results is an array
+
+	results.forEach(function (result) {
+		// ...
+	})
+})
+```
+
+---
+
+### Boolean result.pass
+
+True if the test passed. **No other keys will exist on the result if this value is true.**
+
+---
+
+### Error result.error
+
+The value of the stack property from the `Error` object explaining the reason behind a failure.
+
+---
+
+### String result.message
+
+If an exception was thrown inside the corresponding test, this will equal that Error's `message`. Otherwise, this will be a preformatted message in [SVO form](https://en.wikipedia.org/wiki/Subject%E2%80%93verb%E2%80%93object). More specifically, `${subject}\n${verb}\n${object}`.
+
+As an example, the following test's result message will be `"true\nshould equal\nfalse"`.
+
+```javascript
+o.spec("message", function () {
+	o(false).equals(true)
+})
+```
+
+If you specify an assertion description, that description will appear two lines above the subject.
+
+```javascript
+o.spec("message", function () {
+	o(false).equals(true)("Candyland") // result.message === "Candyland\n\ntrue\nshould equal\nfalse"
+})
+```
+
+---
+
+### String result.context
+
+A `>`-separated string showing the structure of the test specification.
+In the below example, `result.context` would be `testing > rocks`.
+
+```javascript
+o.spec("testing", function () {
+	o.spec("rocks", function () {
+		o(false).equals(true)
+	})
+})
+```
+
+
+
 ---
 
 ## Goals
 
 - Do the most common things that the mocha/chai/sinon triad does without having to install 3 different libraries and several dozen dependencies
 - Disallow configuration in test-space:
 	- Disallow ability to pick between API styles (BDD/TDD/Qunit, assert/should/expect, etc)
-	- Disallow ability to pick between different reporters
```

Comment:
Let's reword this line & move it to the bottom instead of removing it entirely. Maybe something like

> - Provide a default simple reporter